src/ResourceManager/Profile/Commands.Profile/StartupScripts/AzError.ps1

```diff
@@ -0,0 +1,37 @@
+﻿
+$pathToInstallationChecks = Join-Path (Join-Path $HOME ".Azure") "AzInstallationChecks.json"
+if (!(Test-Path $pathToInstallationChecks))
```

Comment:
We should also add in an environment variable to skip these checks, so that those who don't have access to a user directory can avoid this extra overhead